ui/src/logic/utils.ts

```diff
@@ -383,9 +383,9 @@ export function hasKeys(obj: Record<string, unknown>) {
 }
 
 export const IMAGE_REGEX =
-  /(\.jpg|\.img|\.png|\.gif|\.tiff|\.jpeg|\.webp|\.webm|\.svg)$/i;
-export const AUDIO_REGEX = /(\.mp3|\.wav|\.ogg|\.m4a)$/i;
-export const VIDEO_REGEX = /(\.mov|\.mp4|\.ogv)$/i;
+  /(\.jpg|\.img|\.png|\.gif|\.tiff|\.jpeg|\.webp|\.webm|\.svg)(?:\?.*)?$/i;
+export const AUDIO_REGEX = /(\.mp3|\.wav|\.ogg|\.m4a)(?:\?.*)?$/i;
+export const VIDEO_REGEX = /(\.mov|\.mp4|\.ogv)(?:\?.*)?$/i;
```

Comment:
We don't use the capture for anything so a non-capturing group isn't strictly necessary here

Reply:
Cool, I had a look at the downstream usages but wasn't sure whether something might have wanted it, so I figured safest to keep it non-capturing. Apparently they can also perform better, according to https://stackoverflow.com/a/33250820 (though for expressions like these I suspect the difference is negligible).

Happy to change it if preferred though?